ribs/emitters/_iso_line_emitter.py

```diff
@@ -46,15 +46,17 @@ class IsoLineEmitter(EmitterBase):
     def __init__(self,
                  archive,
                  x0,
-                 iso_sigma=0.01,
+                 iso_sigma0=0.01,
+                 iso_sigma=None,
```

Comment:
Let's keep `iso_sigma`, and let's make `iso_sigma0` be called `sigma0`. Then `sigma0` will default to `iso_sigma` if not passed in.

Reply:
Also, add these params to the argument docstring.